simulator-ui/src/main/resources/static/src/app/components/about/about.ts

```diff
@@ -11,12 +11,13 @@ export class AboutComponent implements OnInit {
     simulatorInfo: SimulatorInfo;
 
     constructor(private appInfoService: AppInfoService) {
-        this.appInfoService.getAppInfo().subscribe(
-            appInfo => this.simulatorInfo = appInfo.simulator,
-            error => console.log(error),
-            () => console.log("Info has been retrieved: simulator name = '%s', domain = '%s', version = '%s'",
+        this.appInfoService.getAppInfo().subscribe({
+            next:appInfo => this.simulatorInfo = appInfo.simulator,
```

Comment:
this is not very readable.. applis to all subsequent lines.
```suggestion
            next: (appInfo) => this.simulatorInfo = appInfo.simulator,
```

Reply:
In the newer versions, you now have to add ```next:/error:/complete:``` and surround them with curly brackets.